src/components/LogView/index.js

```diff
@@ -1,6 +1,5 @@
-import React from 'react';
-import ReactList from 'react-list';
 import Highlighter from 'react-highlight-words';
+import React from 'react';
```

Comment:
As a convention, let's keep `import React` at the top of our js files. ReactList is still required in this file.